src/Io/MultipartParser.php

```diff
@@ -82,7 +82,7 @@ public function __construct($uploadMaxFilesize = null, $maxFileUploads = null)
             $this->maxInputNestingLevel = (int)$var;
         }
 
-        $this->uploadMaxFilesize = $uploadMaxFilesize === null ? $this->iniUploadMaxFilesize() : (int)$uploadMaxFilesize;
+        $this->uploadMaxFilesize = $uploadMaxFilesize === null ? Util::iniSizeToBytes(ini_get('upload_max_filesize')) : Util::iniSizeToBytes($uploadMaxFilesize);
```

Comment:
This can be simplified by moving this out of the ternary. Also, needs tests?

Reply:
Updated a test to test this change